config.ru

```diff
@@ -1,3 +1,12 @@
 # frozen_string_literal: true
+require_relative 'lib/cleanup/cleanup.rb'
+
+at_exit do
+  cleanup = Cleanup.new
+  cleanup.shutdown
```

Comment:
Can any exceptions be thrown from this?  Or can any indication be made on the number of containers still running on the host (hopefully zero).